example/ios/example.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just a doubt! Why .xcworlspacedata and this plist files are included in this PR? Previously they were not needed 👀

Reply:
It's created by Xcode automatically you can check. on [stackOverflow](https://stackoverflow.com/questions/50367896/ideworkspacechecks-plist-file-suddenly-appear-after-updated-xcode)